src/app/component/ProductDetails/ProductDetails.component.js

```diff
@@ -65,6 +67,52 @@ class ProductDetails extends Component {
         );
     }
 
+    renderReviewSummary() {
+        const {
+            product: { review_summary, url_key },
+            product,
+            areDetailsLoaded,
+            configurableVariantIndex
+        } = this.props;
+
+        if (areDetailsLoaded) {
+            const linkTo = url_key
+                ? {
+                    pathname: `/product/${ url_key }`,
+                    state: { product, configurableVariantIndex },
+                    search: `?variant=${ configurableVariantIndex }`,
+                    hash: '#reviews'
+                }
+                : undefined;
+
+            if (review_summary.review_count) {
+                const reviewText = review_summary.review_count === 1 ? 'Review' : 'Reviews';
```

Comment:
Seems like you use it twice, please move pluralization function to helpers.

Reply:
Just to be clear - I should create new Review helper under `util` and add a function for this there?